projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/publicize-connection-test-results.php

```diff
@@ -90,6 +90,9 @@ public function get_item_schema() {
 	 * @return WP_REST_Response suitable for 1-page collection
 	 */
 	public function get_items( $request ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
+		$item        = array();
+		$test_result = array();
```

Comment:
Again, this seems to be hiding a real logic error.

Reply:
Yeah, you're right. There's a a few lines of code that is supposed to be in a foreach loop above it that was missed. Updating.